src/inspector_socket_server.h

```diff
@@ -30,17 +30,27 @@ class SocketServerDelegate {
   virtual void ServerDone() = 0;
 };
 
+// HTTP Server, writes messages requested as TransportActions, and responds
+// to HTTP requests and WS upgrades.
 class InspectorSocketServer {
  public:
   using ServerCallback = void (*)(InspectorSocketServer*);
   InspectorSocketServer(SocketServerDelegate* delegate,
                         const std::string& host,
                         int port,
                         FILE* out = stderr);
+  // XXX(sam) why is loop not passed in ctor? Its fixed and known at that time.
```

Comment:
This is the part I want to cleanup the most. Did a couple of half-hearted attempts but never achieved the desired result. It is historical (this code started as a fork of old Debug thread) and also because creation and start were separate at some point (this code was a part of the inpsector Agent).